lib/src/main/kotlin/com/swmansion/starknet/data/serializers/JsonRpcTransactionReceiptPolymorphicSerializer.kt

```diff
@@ -9,7 +9,7 @@ internal object JsonRpcTransactionReceiptPolymorphicSerializer :
     JsonContentPolymorphicSerializer<TransactionReceipt>(TransactionReceipt::class) {
     override fun selectDeserializer(element: JsonElement): DeserializationStrategy<out TransactionReceipt> {
         val jsonElement = element.jsonObject
-        val isPending = "status" !in jsonElement
+        val isPending = "status" !in jsonElement && "execution_status" !in jsonElement
```

Comment:
is this a new property in rpc? what does it indicate?

Reply:
Should we have isPending variable if there are no pending transactions?

Reply:
You are correct, do you think we should check for both or just one?

Reply:
You're right, and there's currently no test that that checks if pending receipt is correctly deserialized, so I'll add one as well.